### DISCUSSION TITLE
Like Any Support in Diesel 2.*

### DISCUSSION THREAD

**Discussion by geckoxx** [diesel-rs/diesel#3631] [Q&A] [answered]
In Diesel 1 it is possible to use `any` with `like` or `ilike`:
`.filter(name.ilike(diesel::dsl::any(subquery)))`

But with Diesel 2 `diesel::dsl::any` got deprecated.
Is there a way to to use `like any` without using a deprecated function?

**Comment by weiznich** [comment, marked as answer]
Duplicate of https://github.com/diesel-rs/diesel/issues/3360
